packages/deploy/src/validator.ts

```diff
@@ -53,6 +53,7 @@ export function parseAndValidate(input: string): {
           pushUniqueKey(workflow, (workflow as any).key.value);
 
           validateJobs((workflow as any).value);
+          // validateEdges((workflow as any).value);
```

Comment:
Can we have a comment to say why this is commented out? Looks kind of important....

Reply:
I've removed it, it _was_ important! :sweat_smile: